Sources/TuistCache/ContentHashing/CoreDataModelsContentHasher.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import TuistCore
+
+public protocol CoreDataModelsContentHashing {
+    func hash(coreDataModels: [CoreDataModel]) throws -> String
+}
+
+public final class CoreDataModelsContentHasher: CoreDataModelsContentHashing {
+    private let contentHasher: ContentHashing
+
+    // MARK: - Init
+
+    public init(contentHasher: ContentHashing = ContentHasher()) {
+        self.contentHasher = contentHasher
+    }
+
+    // MARK: - CoreDataModelsContentHashing
+    
+    public func hash(coreDataModels: [CoreDataModel]) throws -> String {
```

Comment:
As a future optimization we should keep a dictionary with the already-hashed elements, and read the values from there if they have already been hashed. It can just be `[Hashable: String]`. We don't need to do it as part of this PR though.

Reply:
I've done it, but only for files, I think that hashing a string should be fast enough that doesn't need a cache. wdyt?

Reply:
nvm, I'm going to iterate on this asap